### DISCUSSION TITLE
Pet Listings Team Staff (5/5)

### DISCUSSION THREAD

**Discussion by rukitoyordan** [uprm-inso4101-2023-2024-s2/semester-project-rumaradoptionapp#4] [Pet Listings Team]
### Pet Listings Team for Milestone No.1 
**Main Manager**
- Evand Sanchez ()

**Team Leader**
- John P. Santiago Ruiz (@JohnSantiagoRuiz)

**Developers**
- Diego O. Robles Amengual (@diegorobles2)
- Hector G. Ramos Cortes (@cxprss)
- Christian N. Rodriguez Figueroa (@Christian2101)
- Alejandro J. Rivera Batista (@Alej24)
- John P. Santiago Ruiz (@JohnSantiagoRuiz)

**Comment by rukitoyordan** [comment]
@diegorobles2 @cxprss @Christian2101 @Alej24 @JohnSantiagoRuiz, por favor dejenme saber quienes de ustedes estan interesados en ser Team Leader! Recuerden que se deben reunir e ir creando tasks en el tab de "Issues" lo antes posible. Entiendo que pueden tener problemas con la electricidad al momento asi que solo intenten lograr eso para que los managers podamos ir asignando puntos y objetivos!

**Reply by JohnSantiagoRuiz** [reply]
I’ll take team leader.

**Comment by JohnSantiagoRuiz** [comment]
In order to have a place where we can easily communicate, please join the WhatsApp group with the link provided below.

WhatsApp Link: https://chat.whatsapp.com/DS35wXmg90H7WlEq4s9Ubx
